.github/ISSUE_TEMPLATE/Feature_Request.md

```diff
@@ -11,7 +11,6 @@ about: You have a neat idea that should be implemented? 🎩
 |------------ | ------
 | New Feature | yes
```

Comment:
This is tautological

Reply:
I'd also remove the RFC question since we don't have a formal process for those.